client/src/App.js

```diff
@@ -19,8 +19,10 @@ const App = () => {
   useEffect(() => {
     if (useMock === true) {
       store.dispatch.data.fetchRivers(mockRivers);
+      store.dispatch.editing.fetchRivers(mockRivers);
```

Comment:
This reducer is the same as in `data` model. Did you create it in advance to something?

Reply:
Good catch! Meant to take that out, it was just me trying something that I decided not to do.